src/XliffTasks.Tests/XliffDocumentTests.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+
+namespace XliffTasks.Tests
+{
+    public class XliffDocumentTests
+    {
+        [Fact]
+        public void NewDocument()
```

Comment:
I think we should name this test better, from just reading it I have no idea what you are trying to assert in your test.